skbeam/core/correlation.py

```diff
@@ -754,7 +774,12 @@ def _validate_and_transform_inputs(num_bufs, num_levels, labels):
     num_pixels = np.bincount(label_array)[1:]
 
     # Convert from num_levels, num_bufs to lag frames.
-    tot_channels, lag_steps = multi_tau_lags(num_levels, num_bufs)
+    tot_channels, lag_steps, dict_lag = multi_tau_lags(num_levels, num_bufs)
+
+    # these norm and lev_len will help to find the one time correlation
+    # normalization norm will updated when there is a bad image
+    norm = {key: [0] * len(dict_lag[key]) for key in list(dict_lag.keys())}
```

Comment:
Don't call `list` on these keys.  It is an unnecessary step since you are just iterating over this.  Change this to `for key in dict_lag.keys()`